charts/super-agent/templates/_helm-release.yaml

```diff
@@ -43,4 +43,7 @@ spec:
   rollback:
     {{- . | toYaml | nindent 4 }}
   {{- end }}
+  uninstall: # This is setup explicitly since executing hooks and waiting for uninstall termination is needed to remove super-agent managed resources
```

Comment:
have we changed the approach, right?
The idea is to simply leverage the chart and wait I guess

Reply:
Yes we have! 
Since we can wait for the `super-agent-deployment` uninstall, there is no need to repeat the super-agent managed resources deletion in this chart.